freqtrade/optimize/hyperopt.py

```diff
@@ -118,11 +119,14 @@ def __init__(self, config: Config) -> None:
 
         # Use max_open_trades for hyperopt as well, except --disable-max-market-positions is set
         if self.config.get('use_max_market_positions', True):
-            self.max_open_trades = self.config['max_open_trades']
+            self.max_open_trades = self.config['max_open_trades'] \
+                if self.config['max_open_trades'] != float('inf') else -1
         else:
             logger.debug('Ignoring max_open_trades (--disable-max-market-positions was used) ...')
             self.max_open_trades = 0
 
+        print("Strategy max open trades", self.max_open_trades)
```

Comment:
don't think we should print here ... seems like a debug remain?

Reply:
Yes, I'm removing it :D